screens/Home/ViewVcModal.tsx

```diff
@@ -60,7 +55,23 @@ export const ViewVcModal: React.FC<ViewVcModalProps> = props => {
       onDismiss={props.onDismiss}
       headerTitle={t('title')}
       testID="idDetailsHeader"
-      headerElevation={2}>
+      headerElevation={2}
+      arrowLeft={<Icon name={''} />}
```

Comment:
Instead of passing empty icon, can't we not pass the arrowLeft param and handle it accordingly.

Reply:
@vijay151096 This `arrowLeft` prop change will be from my PR. It's the same prop. related changes in Modal.tsx were added from @srikanth716 PR (means this PR).